model-providers/jlama/runtime/src/main/java/io/quarkiverse/langchain4j/jlama/JlamaChatModel.java

```diff
@@ -8,12 +8,14 @@
 import java.util.Optional;
 import java.util.UUID;
 
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
```

Comment:
We usually use the JBoss Logging version

Reply:
Fixed.